src/Essentials/src/SecureStorage/SecureStorage.android.cs

```diff
@@ -12,50 +12,46 @@ partial class SecureStorageImplementation : ISecureStorage
 
 		Task<string> PlatformGetAsync(string key)
 		{
-			return Task.Run(() =>
+			string data = null;
+
+			try
 			{
-				try
-				{
-					lock (locker)
-					{
-						return GetEncryptedSharedPreferences().GetString(key, null);
-					}
-				}
-				catch (AEADBadTagException)
+				lock (locker)
 				{
-					// TODO: Use Logger here?
-					System.Diagnostics.Debug.WriteLine($"Unable to decrypt key, {key}, which is likely due to an app uninstall. Removing old key and returning null.");
-					Remove(key);
-
-					return null;
+					data = GetEncryptedSharedPreferences().GetString(key, null);
 				}
-				catch (Java.Lang.SecurityException)
-				{
-					// TODO: Use Logger here?
-					System.Diagnostics.Debug.WriteLine($"Unable to decrypt key, {key}, which is likely due to key corruption. Removing old key and returning null.");
-					Remove(key);
+			}
+			catch (AEADBadTagException)
+			{
+				// TODO: Use Logger here?
+				System.Diagnostics.Debug.WriteLine($"Unable to decrypt key, {key}, which is likely due to an app uninstall. Removing old key and returning null.");
+				Remove(key);
+			}
+			catch (Java.Lang.SecurityException)
+			{
+				// TODO: Use Logger here?
+				System.Diagnostics.Debug.WriteLine($"Unable to decrypt key, {key}, which is likely due to key corruption. Removing old key and returning null.");
+				Remove(key);
+			}
 
-					return null;
-				}
-			});
+			return Task.FromResult(data);
 		}
 
 		Task PlatformSetAsync(string key, string data)
 		{
-			return Task.Run(() =>
+			lock (locker)
 			{
-				lock (locker)
-				{
-					using var editor = GetEncryptedSharedPreferences().Edit();
+				using var editor = GetEncryptedSharedPreferences().Edit();
 
-					if (data == null)
-						editor.Remove(key);
-					else
-						editor.PutString(key, data);
+				if (data == null)
```

Comment:
```suggestion
				if (data is null)
```

Reply:
@mandel-macaque is there an .editorconfig rule for this? If we turn it on, the autoformat bot will automatically fix it on PRs.

Reply:
Pushed this change, but if there is a way to enable it more broadly maybe that can be done separately.

Reply:
we should def look into that cause a lot of devs in the project are doing it.

Reply:
Done in #14002